shared/domains/chelonia/internals.js

```diff
@@ -253,7 +253,22 @@ export default (sbp('sbp/selectors/register', {
       if (latest !== recent) {
         console.debug(`[chelonia] Synchronizing Contract ${contractID}: our recent was ${recent || 'undefined'} but the latest is ${latest}`)
         // TODO: fetch events from localStorage instead of server if we have them
-        const events = await sbp('chelonia/out/eventsSince', contractID, recent || contractID)
+        const events = await sbp('chelonia/out/eventsAfter', contractID, recent || contractID)
+        // checks if the list of events consist of latest event
+        // TODO: if we use findLastIndex, it will be more clean
+        //       but it needs upgrade Cypress version to 9.7.0 which is of bad performance
+        //       https://docs.cypress.io/guides/references/changelog#9-7-0
+        //       https://github.com/cypress-io/cypress/issues/22868
```

Comment:
```suggestion
        // TODO: using findLastIndex, it will be more clean but it needs Cypress 9.7+ which has bad performance
        //       https://docs.cypress.io/guides/references/changelog#9-7-0
        //       https://github.com/cypress-io/cypress/issues/22868
```